src/debug.rs

```diff
@@ -1,8 +1,10 @@
 //! The native debug interface exposed to the ewasm contract. These functions are for testing
 //! purposes only. On a live VM, any bytecode trying to import these symbols will be rejected.
 
-use crate::types::*;
+use crate::types::StorageKey;
 
+/// The native interface for debugging functions. Use the wrappers unless you know what you're
```

Comment:
These are not exported so not sure they need to be documented. At least "Use the wrapper unless you know what you're doing" is pointless.